### PR TITLE
[10.0] [FIX]  Fix bug when creating a chopinvader categoy from category

### PR DESCRIPTION
**Pull request by sebastienbeau**
try to fix https://github.com/shopinvader/odoo-shopinvader/pull/350

### CODE REVIEW

File: base_url/models/abstract_url.py [hunk 1]
```diff
@@ -239,6 +239,10 @@ def _sync_urls(self):
 
     @api.model
     def create(self, value):
+        # Creating a binding from the main record set the recompute env to
+        # False and broke the implementation of url sync
+        # need to force it to True
+        self.env.all.recompute = True
```
Comment (lmignon) [inline base_url/models/abstract_url.py:245]: @sebastienbeau I'm pretty sure that this change could lead to performance issue.... Moreover, why do we have a problem with categories and not with products...

**Comment by rousseldenis** [comment]
@sebastienbeau

File: base_url/models/abstract_url.py (hunk 1 above)
Comment (sebastienbeau) [reply base_url/models/abstract_url.py:245]: Hi @lmignon I have refactored this PR and created a new one.
https://github.com/shopinvader/odoo-shopinvader/pull/607

**Comment by sebastienbeau** [comment]
replaced by https://github.com/shopinvader/odoo-shopinvader/pull/607
with a better implementation

**Pull request closed without merging**